packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv6/ipv6.constant.js

```diff
@@ -0,0 +1,53 @@
+import {
+  TRACKING_PREFIX,
+  DASHBOARD,
```

Comment:
Same thing as in ipv4 constant file, use directly the constant defined in parent within the controller
```suggestion
```